pyaedt/desktop.py

```diff
@@ -65,6 +65,42 @@
     settings.use_grpc_api = True
 
 
+def launch_aedt_in_lsf(non_graphical, port):  # pragma: no cover
+    """Launch AEDT in Lsf in GRPC mode."""
```

Comment:
```suggestion
    """Launch AEDT in LSF in GRPC mode."""
```
Changing to all capital letters as most of the entries seem to put it there. Should we define what LSF stands for?